### PR TITLE
add a post method and two more public methods

### PR DESCRIPTION
**Pull request by gergnz**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
-    version='0.2',
+    version='0.3',
     author='Michael Still',
     author_email='mikal@stillhq.com',
     description='A python library to interact with the Aussie Broadband customer portal',
     long_description=long_description,
     long_description_content_type='text/markdown',
     url='https://www.madebymikal.com/aussiebb',
     packages=setuptools.find_packages(),
+    install_requires=['requests'],
```
Comment (mikalstill) [inline setup.py:16]: Isn't having requests in requirements.txt sufficient?

**Review by mikalstill — COMMENTED** [review]
Apart from my one comment this looks good to me, thanks for the contribution!

File: setup.py (hunk 1 above)
Comment (gergnz) [reply setup.py:16]: I had to manually install it.

I'm not a python guru, but I found this one (to help in my own learnings too): https://packaging.python.org/discussions/install-requires-vs-requirements/

I think the most relevant sentence for us in this situation is this one:
```
Whereas install_requires metadata is automatically analyzed by pip during an install, requirements files are not, and only are used when a user specifically installs them using pip install -r.
```

Hope that helps. Thanks so much for this. Gave me a starting point for my project.

**Comment by mikalstill** [comment]
Sorry, I lost this in my inbox, thanks for the contribution.

**Comment by gergnz** [comment]
sorry to bug you again, do you think you can get this published on pypi? Thanks.

**Pull request merged**